core/chains/evm/config/toml/defaults/L3X_Mainnet.toml

```diff
@@ -0,0 +1,18 @@
+ChainID = '12324'
+ChainType = 'arbitrum'
+FinalityTagEnabled = true
+FinalityDepth = 10
+LinkContractAddress = '0x79f531a3D07214304F259DC28c7191513223bcf3'
+# Produces blocks on-demand
+NoNewHeadsThreshold = '0'
+OCR.ContractConfirmations = 1
+LogPollInterval = '10s'
+
+[GasEstimator]
+Mode = 'Arbitrum'
```

Comment:
Have we tested whether the existing precompiles work or not?

Reply:
yes, we've tested this. called the RPC with the precompile address with the correct 4 bytes